src/@next/Input/Input.tsx

```diff
@@ -63,7 +91,30 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
         suffixWidth={suffixWidth}
       >
         <Prefix />
-        <StyledInput ref={inputRef} disabled={disabled} {...props} />
+        <StyledInput
+          ref={inputRef}
+          placeholder={isPlaceholderFloating ? undefined : selectedPlaceholder}
+          borderRadius={borderRadius}
+          border={border}
+          height={height}
+          disabled={disabled}
+          {...props}
+        />
+        {isPlaceholderFloating && (
+          <FloatingLabel
+            data-testid="textarea-label"
+            fontSize={floatingFontSize}
+          >
+            {required ? (
+              <>
+                {placeholder}
+                <AsteriskIcon>*</AsteriskIcon>
+              </>
+            ) : (
+              placeholder
+            )}
+          </FloatingLabel>
+        )}
```

Comment:
Sorry I missed this. Can you help update this too?
```suggestion
        {isPlaceholderFloating && (
          <FloatingLabel data-testid="textarea-label">
            {placeholder}
            {required && <AsteriskIcon>*</AsteriskIcon>}
          </FloatingLabel>
        )}
```